.schema/config.schema.json

```diff
@@ -25,6 +25,30 @@
       "minimum": 1,
       "maximum": 65535
     },
+    "socket": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "Sets the permissions of the unix socket",
+      "properties": {
+        "owner": {
+          "type": "string",
+          "description": "Owner of unix socket. If empty, the owner will be the user running hydra.",
+          "default": ""
```

Comment:
```suggestion
          "default": 755,
          "max": 777,
          "min": 000,
```

Reply:
I'm not sure you want this. owner and group should be strings indicating the names of the owner and group. On the other hand, I changed the default mode to 755. It's a shame json doesn't support octal notation (or comments)

Reply:
Everything else fixed

Reply:
Ohhh, my bad - makes sense!